### PR TITLE
(chore)cleanup gradle

### PR DESCRIPTION
**Pull request by spbolton**
Remove old code no longer being used after gradle and relocate integration test files into the correct maven location into the dotcms-integration folder. 

[https://github.com/dotCMS/internal-infrastructure/issues/542](url)

### CODE REVIEW

File: dotcms-integration/pom.xml [hunk 1]
```diff
@@ -291,43 +289,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
```
Comment (spbolton) [inline dotcms-integration/pom.xml:294]: No longer need this block now sources and resources are in the standard location

**Review by victoralfaro-dotcms — APPROVED** [review]

File: dotCMS/src/assembly/descriptor.xml
Comment (wezell) [inline dotCMS/src/assembly/descriptor.xml:1]: Don't think we need this change now, do we?

File: dotCMS/src/main/docker/original/docker-descriptor.xml
Comment (wezell) [inline dotCMS/src/main/docker/original/docker-descriptor.xml:1]: do we need this or was it taken care of in #27412

**Review by wezell — APPROVED** [review]
Too big to review properly - we will need to merge and test.

File: dotCMS/src/assembly/descriptor.xml
Comment (spbolton) [reply dotCMS/src/assembly/descriptor.xml:1]: This is the equivalent of the docker-descriptor used to package tomcat with the app in the deployment zip package.

Comment (spbolton) [reply dotCMS/src/assembly/descriptor.xml:1]: Deployment script no longer required so removed

File: dotCMS/src/main/docker/original/docker-descriptor.xml
Comment (spbolton) [reply dotCMS/src/main/docker/original/docker-descriptor.xml:1]: taken care of in #27412

**Pull request merged**